### ISSUE TITLE
dock_64 keep on being lost

### ISSUE THREAD

**Issue by bryanmartinph** [mydockfinder/mydockfinder#300]
I've installed mydockfinder earlier this day, it's unbelievably awesome. But one of my problem is the dock_64 application file inside the MyDock folder keep on being lost, thus it causes the mydockfinder to fail to launch, this is so scary for me because I can't access the taskbar as it's not showing up. I'm new to this mydockfinder app, hoping to receive an answer!

**Comment by mydockfinder** [comment]
杀毒软件，如果是win10就是Windows安全中心，添加信任排除项就可以了，个人做的软件没有数字签名会被报病毒的




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mydockfinder/mydockfinder-for-Win10-Win11"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月7日(星期二) 晚上10:16
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[mydockfinder/mydockfinder-for-Win10-Win11] dock_64 keep on being lost (Issue #300)





 
I've installed mydockfinder earlier this day, it's unbelievably awesome. But one of my problem is the dock_64 application file inside the MyDock folder keep on being lost, thus it causes the mydockfinder to fail to launch, this is so scary for me because I can't access the taskbar as it's not showing up. I'm new to this mydockfinder app, hoping to receive an answer!
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by bryanmartinph** [comment]
Dude, thanks a lot! I found that it's Avast Antivirus moving the file to the quarantine chest, what's strange here is I haven't received any notification about it. But one last thing, mydockfinder is safe right?

**Comment by mydockfinder** [comment]
是的，因为我没有数字证书，放心吧，这个软件已经五年了，而且在steam上收费，肯定是安全的，要是有病毒或者其他的steam审核也过不了，而且我也就不收费了，免费的扩散范围更大




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年12月7日(星期二) 晚上11:42
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock_64 keep on being lost (Issue #300)






哥们，多谢了！我发现是 Avast 杀毒把文件移到隔离箱里奇怪的是我没有收到任何通知。但最后一件事， my dock finder 是安全的吧？

—
你收到这个是因为你评论了。
直接回复此邮件，在 GitHub 上查看或取消订阅.
GitHub Mobile 上的分流通知监督办或机器人.

**Comment by bryanmartinph** [comment]
I've already moved the mydock folder to the Avast's scan exceptions, and the issue is now fixed! My dock finder is working flawlessly now! But just a question, if a new update of mydockfinder came out, how am i supposed to update the app without losing my preferences? Sorry if my question might sound dumb as I'm just new to this 😅

**Comment by mydockfinder** [comment]
每次更新是不会重置配置文件的，只不过现在还不支持切换主题，所以替换的图片会被重置，下一个大版本更新就加入切换主题了




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年12月9日(星期四) 晚上10:25
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock_64 keep on being lost (Issue #300)





 
I've already moved the mydock folder to the Avast's scan exceptions, and the issue is now fixed! My dock finder is working flawlessly now! But just a question, if a new update of mydockfinder came out, how am i supposed to update the app without losing my preferences? Sorry if my question might sound dumb as I'm just new to this 😅
 
&mdash;
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by bryanmartinph** [comment]
No, but I mean how am I supposed to update the app? Do I need to replace the files inside the mydock folder manually or I have to install steam?

**Comment by mydockfinder** [comment]
需要安装steam，更新都是从steam里面更新的




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年12月9日(星期四) 晚上10:37
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock_64 keep on being lost (Issue #300)





 
No, but I mean how am I supposed to update the app? Do I need to replace the files inside the mydock folder manually or I have to install steam?
 
&mdash;
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by bryanmartinph** [comment]
Oh, I see. Thanks for your response, your app is exceptionally awesome! I've read that your going to make mydockfinder available on the Microsoft Store soon too. I'll wait for that, since updates will be much convenient! 😄

**Comment by mydockfinder** [comment]
但是商店上架还要等一段时间，我起码要等steam创意工坊的功能完成，最少也得两到三个月




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年12月9日(星期四) 晚上11:36
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock_64 keep on being lost (Issue #300)





 
Oh, I see. Thanks for your response, your app is exceptionally awesome! I've read that your going to make mydockfinder available on the Microsoft Store soon too. I'll wait for that, since updates will be much convenient! 😄
 
&mdash;
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by bryanmartinph** [comment]
Okay, that's nice! Do you also know how can I add more apps on the launchpad? Dragging shortcuts to the launchpad doesn't work.

**Comment by mydockfinder** [comment]
dock上的launchpad图标，右键添加图标




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年12月10日(星期五) 下午4:52
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock_64 keep on being lost (Issue #300)





 
Okay, that's nice! Do you also know how can I add more apps on the launchpad? Dragging shortcuts to the launchpad doesn't work.
 
&mdash;
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by imbryanmartin** [comment]
No, I mean, how can I add more apps on the launchpad itself? Or that feature is not yet available?

**Comment by bryanmartinph** [comment]
Oh, I got it now! Thank you much! Sorry for asking too much! You guys really did well with mydockfinder! Waiting for more exciting update!

**Comment by mydockfinder** [comment]
不可以，谢谢支持



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年12月10日(星期五) 晚上7:45
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] dock_64 keep on being lost (Issue #300)





 
Oh, I got it now! Thank you much! Sorry for asking too much! You guys really did well with mydockfinder! Waiting for more exciting update!
 
&mdash;
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
